### PR TITLE
[143] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by lorlahdey**
image7

### CODE REVIEW

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                <li><a href="#">Garden</a></li>
+                <li><a href="#">Vases</a></li>
+            </ul>
+        </nav>
+    </div>
+</header>
+    <!-- Header ends -->
+
+
+    <!-- Contact Us Form -->
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:27]: Remove whitespaces

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                <div>
+                    <label for="lastname"><strong>Last Name:</strong></label>&nbsp;&nbsp;&nbsp;&nbsp;
+                    <input type="text" name="lastname" id="lastname">
+                </div><br>
+                <div>
+                    <label for="email"><strong>Email:</strong></label>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                    <input type="email" name="email" id="email">                    
+                </div><br>
+                <div>
+                    <label><strong>Subject:</strong></label>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:46]: Kindly remove the extra characters after the closing tag. The section is also not properly formatted.

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                <div>
+                    <input type="submit" value="Submit" onclick="alert('Thank you, we will response shortly.')">
+                </div>
+            </fieldset>
+        </form>
+    </div>
+</section>
+
+
+    <!-- Footer Begins -->
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:67]: Remove extra whitespaces

File: submissions/lawalLolade/html-forms/contactFormPage.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+<footer>
+    <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">Events</a></li>
+    </ul>
+    <p> &copy; 2020 Flawa. </p>
+</footer>
+    <!-- Footer Ends -->
+
```
Comment (Ireti203) [inline submissions/lawalLolade/html-forms/contactFormPage.html:77]: Remove extra whitespaces

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by lorlahdey** [comment]
Ireti

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**